make_deb

```diff
@@ -21,68 +21,57 @@
 
 set -e
 
-architecture="none"
 plugin_type="south"
 plugin_name="dht"
 plugin_install_dirname="dht"
-usage="$(basename "$0") {x86|arm} [help|clean|cleanall]
+usage="$(basename "$0") [help|clean|cleanall]
 This script is used to create the Debian package of FoglAMP C++ '${plugin_name}' ${plugin_type} plugin
 Arguments:
  help     - Display this help text
- x86      - Build an x86_64 package
- arm      - Build an armv7l package
  clean    - Remove all the old versions saved in format .XXXX
  cleanall - Remove all the versions, including the last one"
 
 GIT_ROOT=`pwd`    # The script must be executed from the root git directory
-if [ $# -gt 0 ]; then
-    for i in "$@"
-    do
-        case "$i" in
-          x86)
-              architecture="x86_64"
-              shift
-              ;;
-          arm)
-              architecture="armhf"
-              shift
-              ;;
-          clean)
-              echo -n "Cleaning the build folder from older versions..."
-              find "${GIT_ROOT}/packages/build" -maxdepth 1 | grep '.*\.[0-9][0-9][0-9][0-9]' | xargs rm -rf
+
+for i in "$@"
+do
+    case "$i" in
+      clean)
+          echo -n "Cleaning the build folder from older versions..."
+          find "${GIT_ROOT}/packages/build" -maxdepth 1 | grep '.*\.[0-9][0-9][0-9][0-9]' | xargs rm -rf
+          echo "Done."
+          exit 0
+          ;;
+      cleanall)
+          if [ -d "${GIT_ROOT}/packages/build" ]; then
+              echo -n "Cleaning the build folder..."
+              rm -rf ${GIT_ROOT}/packages/build/*
               echo "Done."
-              shift
-              ;;
-          cleanall)
-              if [ -d "${GIT_ROOT}/packages/build" ]; then
-                  echo -n "Cleaning the build folder..."
-                  rm -rf ${GIT_ROOT}/packages/build/*
-                  echo "Done."
-              else
-                  echo "No build folder, skipping cleanall"
-              fi
-              shift
-              ;;
-          help)
-              echo "${usage}"
-              exit 0
-              ;;
-          *)
-              echo "Unrecognized option: $i"
-             exit 1
-            ;;
-      esac
-    done
-else
-  echo "See help and select the architecture to use, *x86* or *arm*"
-  exit 1
-fi
+          else
+              echo "No build folder, skipping cleanall"
+          fi
+          exit 0
+          ;;
+      help)
+          echo "${usage}"
+          exit 0
+          ;;
+      *)
+          echo "Unrecognized option: $i"
+         exit 1
+        ;;
+  esac
+done
+
 
 # If the architecture has not been defined, then the script is complete
-if [[ "$architecture" == "none" ]]; then
-  exit 0
+if [ "$(dpkg --print-architecture)" != "armhf" ]; then
```

Comment:
I would suggest `if [ "$(dpkg --print-architecture)" != "arm*" ];` then to keep it generic for ARM

See conversation on [FOGL-1470](https://scaledb.atlassian.net/browse/FOGL-1470?focusedCommentId=39432&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-39432)

Reply:
We cant keep it generic for the below mentioned reasons:
1. Its not tested on other arm architectures - armel and arm64
2. Even if we test that for 1, we need to provide a Control file for that particular architecture. I am not sure if we want to do that currently.  Note packages/Debian/armhf/DEBIAN/control:Architecture